devel/icu-devel/Portfile

```
@@ -116,10 +116,20 @@ if {${subport} eq ${name} || ${subport} eq "${name}-lx"} {
     test.target     check
     test.args       VERBOSE=1
 
+    # icu4c requires python2.7+ regardless of configure:
+    #   checking for python3...
+    # libxml which is used to compile llvm/clang depends on icu,
+    # and adding MacPorts' python may introduce a dependency loop.
+    # See: https://unicode-org.atlassian.net/browse/ICU-20301
+    # See: https://trac.macports.org/ticket/65870
     if {${os.platform} eq "darwin" && ${os.major} < 11} {
-        depends_build-append    port:python27
+        depends_build-append    port:python27-bootstrap
         license_noconflict      python27
-        configure.python        ${prefix}/bin/python2.7
+        configure.python        ${prefix}/libexec/python27-bootstrap/bin/python2.7
+    } elseif {${os.platform} eq "darwin" && ${os.major} < 19} {
+        configure.python        /usr/bin/python3
```

/usr/bin/python3 does not exist on Darwin < 19. Did you mean:
```suggestion
    } elseif {${os.platform} eq "darwin" && ${os.major} >= 19} {
        configure.python        /usr/bin/python3
```